src/styles/_calendar.scss

```
@@ -0,0 +1,49 @@
+.calendar {
+  margin: auto;
+  display: flex;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+  width: $calendarWidth;
+  gap: 1px;
```

use reuse this value a lot of time, so, create variable for this
```suggestion
  gap: $cellGap;
```

---

Regarding the "month-length" modifier. I've made amends but I'm not quite sure how it needs to be done